src/pepr/operator/reconcilers/package-reconciler.ts

```diff
@@ -66,8 +68,15 @@ export async function packageReconciler(pkg: UDSPackage) {
     await enableInjection(pkg);
 
     // Configure SSO
+    const authserviceClients = getAuthserviceClients(pkg);
+    if (authserviceClients.length > 0) {
+      // for each authservice client, create a default deny policy
+      for (const sso of authserviceClients) {
+        await preApplyDefaultDeny(sso.enableAuthserviceSelector!, pkg, sso.clientId);
+      }
+    }
```

Comment:
I'm good with this as is. I'm still not really sure I see this being common enough of an ask that we should make it the default behavior but I'm happy to see what shakes out with the user experience around this.

I do wonder if we should document this expectation for users though - if authservice/keycloak things failed they should expect to be locked out of their app until they resolve issues with it?